apps/pdb/src/main.cpp

```diff
@@ -0,0 +1,66 @@
+#ifdef TODO
```

Comment:
nit: i think better not to clutter codebase with unfinished stuff unless ur like literally about to follow up with a pr that finishes it or smth.

Reply:
hmm i think there's value to adding skeleton code like here. irdk the best way to do it. this could be on some branch but then we'll prob forget abt it or something. maybe we should make the rule that is fine but only of main.cpps?

Reply:
I think skeleton code is fine. It just needs to be done enough so that it compiles. Like u can have empty functions and stuff with too comments. Just not have the janky idea todo